### DISCUSSION TITLE
2022-03-10 计划转行做【产品经理】了 ~ 蒋继发 (ThaddeusJiang)

### DISCUSSION THREAD

**Discussion by giscus** [ThaddeusJiang/thaddeusjiang.github.io#28] [Announcements]
# 2022-03-10 计划转行做【产品经理】了 ~ 蒋继发 (ThaddeusJiang)

其实这个想法很早以前就有了。不过人嘛，总是患得患失不敢轻易做出改变。

https://thaddeusjiang.com/2022-03-10--ji-hua-zhuan-xing-zuo--chan-pin-jing-li--liao.html

**Comment by czqme** [comment]
Cool，技术转产品是有优势的，引用许式伟在他的课程中关于产品和技术的定义。

「产品是桥，它一端连接了用户需求，一端连接了先进的技术。产品经理是需要有技术高度的，他不一定要深刻了解技术的原理，但是一定要深刻理解新技术的边界。」
「产品经理和架构师其实是一体两面。两者都需要关心用户需求与产品定义。」

**Comment by bestony** [comment]
这一点我和你是比较像的。我们喜欢的是创造，而不是写代码。代码是创造的手段。开发一些小的产品，然后用这些产品去解决切实存在的问题。如果能养活自己就更好了。

**Reply by ThaddeusJiang** [reply]
@bestony  同道中人 🤝

**Comment by exposir** [comment]
cool
